src/Geopilot.Frontend/src/pages/admin/Users.tsx

```diff
@@ -1,9 +1,161 @@
 import { useTranslation } from "react-i18next";
+import { CircularProgress, Stack } from "@mui/material";
+import { AdminGrid } from "../../components/adminGrid/AdminGrid.tsx";
+import { DataRow, GridColDef } from "../../components/adminGrid/AdminGridInterfaces.ts";
+import { ErrorResponse, Organisation, User } from "../../AppInterfaces.ts";
+import { useContext, useEffect, useState } from "react";
+import { useAuth } from "../../auth";
+import { AlertContext } from "../../components/alert/AlertContext.tsx";
+import { PromptContext } from "../../components/prompt/PromptContext.tsx";
 
 export const Users = () => {
   const { t } = useTranslation();
+  const { user } = useAuth();
+  const [users, setUsers] = useState<User[]>();
+  const [organisations, setOrganisations] = useState<Organisation[]>();
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const { showAlert } = useContext(AlertContext);
+  const { showPrompt } = useContext(PromptContext);
 
-  return <>{t("users")}</>;
+  useEffect(() => {
+    if (users && organisations) {
+      setIsLoading(false);
+    }
+  }, [users, organisations]);
+
+  async function loadUsers() {
```

Comment:
FYI/PP: Wir sollten bald eine generelle Fetch-Methode ins Auge fassen, die in etwa das Muster von `loadUsers()`/`loadOrganisations` abdeckt. Das würde einige Wiederholungen sparen. [In der `App`-Klasse hat es bereits so etwas in der Art.](https://github.com/GeoWerkstatt/geopilot/blob/f677a821280cfee9aeaf43c0a0d73c5dab3ea29d/src/Geopilot.Frontend/src/App.tsx#L63-L80)